src/Handler/TagHandler.php

```diff
@@ -83,7 +75,7 @@ public function getTagsHeaderValue()
      */
     public function hasTags()
     {
-        return 0 < count($this->tags);
+        return !empty($this->tags);
```

Comment:
consistency with `if (empty($this->tags)) {`, and perf, but neglectable, so can remove the change.